src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -5137,10 +5142,10 @@ public static final Modifiers recalculateAdjustments(
         Modifiers.evaluatedModifiers("Mummery", Preferences.getString("_mummeryMods")));
 
     // Add modifiers from inventory
-    if (InventoryManager.hasItem(ItemPool.FISHING_POLE)) {
+    if (InventoryManager.getCount(ItemPool.FISHING_POLE) > 0) {
```

Comment:
I think this technically does something slightly different. Does `hasItem` also get the item out of the closet?

Reply:
Yeah, this looked wrong and I fixed it while I was there. `hasItem` checks whatever you have marked as `autoSatisfy`, which is definitely not the right model for "does this item affect me by being in my inventory right now."